LibreNMS/Modules/Wireless.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace LibreNMS\Modules;
+
+use LibreNMS\DB\SyncsModels;
+use LibreNMS\Interfaces\Module;
+use LibreNMS\Interfaces\Polling\WirelessPolling;
+use LibreNMS\OS;
+use LibreNMS\Util\ModuleModelObserver;
+
+class Wireless implements Module
+{
+    use SyncsModels;
+
+    /**
+     * Discover this module. Heavier processes can be run here
+     * Run infrequently (default 4 times a day)
+     *
+     * @param OS $os
+     */
+    public function discover(OS $os)
+    {
+
+    }
+
+    /**
+     * Poll data for this module and update the DB / RRD.
+     * Try to keep this efficient and only run if discovery has indicated there is a reason to run.
+     * Run frequently (default every 5 minutes)
+     *
+     * @param OS $os
+     */
+    public function poll(OS $os)
+    {
+        if ($os instanceof WirelessPolling) {
+            echo "\nWireless Access Points: ";
+            $os->pollWirelessAccessPoints();
```

Comment:
This should return the data from snmp formatted in a standard way (like in an unsaved AccessPoint model so you can sync it)

Then in the common code is where you save data to the DB and RRDs.

Reply:
Thanks, I'll output the data as AccessPoint-models when ready